internal/server/v2/observation/contained_in.go

```diff
@@ -186,6 +186,11 @@ func FetchContainedIn(
 	var err error
 	if store.BtGroup != nil {
 		readCollectionCache := false
+		childPlaces, err = FetchChildPlaces(
+			ctx, store, metadata, httpClient, remoteMixer, ancestor, childType)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
We don't want to always fetch this since the obsCollection collection could already exists from BT cache.